docs/_tooling/extensions/score_source_code_linker/parse_source_files.py

```diff
@@ -0,0 +1,123 @@
+# *******************************************************************************
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+# *******************************************************************************
+import subprocess
+import os
+import argparse
+import collections
+import functools
+import json
+from typing import Optional
+from pathlib import Path
+
+
+TAGS = [
+    "# req-traceability:",
+    "# req-Id:",
```

Comment:
Should be discussed after PR is merged. `#` sounds rather python specific. The space is strange. etc.